<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemPropertyDialog</class>
 <widget class="QDialog" name="ItemPropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Property</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupDb">
     <property name="title">
      <string>Database</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelID">
        <property name="text">
         <string>&amp;ID:</string>
        </property>
        <property name="buddy">
         <cstring>lineID</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFile">
        <property name="text">
         <string>&amp;File:</string>
        </property>
        <property name="buddy">
         <cstring>lineFile</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineFile"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineID"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupDbAddition">
     <property name="title">
      <string>Database Addition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelUrl">
        <property name="text">
         <string>&amp;Url:</string>
        </property>
        <property name="buddy">
         <cstring>lineUrl</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMemo">
        <property name="text">
         <string>&amp;Memo:</string>
        </property>
        <property name="buddy">
         <cstring>plainTextMemo</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="plainTextMemo"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="lineUrl"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupPersonal">
     <property name="title">
      <string>Personal settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbOpenCount"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelOpenCount">
        <property name="text">
         <string>&amp;OpenCount:</string>
        </property>
        <property name="buddy">
         <cstring>sbOpenCount</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemPropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemPropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
